File: Youtube-Learning/CWH_JavaScript/Tutorial-09/factorial.js
let fact = 1;
// let n = prompt("Enter a number to find its factoril")
let n = 5;

n = Number.parseInt(n)
for(let i = 1; i <= n; i++)
{
    fact *= i
    
}
console.log("Factorial of", n , "is", fact )
    

File: Youtube-Learning/02_Python/100dayschallenge/factorial.py
def factorial(n):
    if n == 0:
        return 1
    elif(n<0):
        print("Factorial is not defined for negative numbers.")
        return -1 
    else:
        return n * factorial(n-1)

number = int(input("Enter a number: "))
print("Factorial:", factorial(number))

# n = 5 - 1 = 4 20*3 60*2 120*1 120
